opensrp-unicef-tunisia/src/test/java/org/smartregister/uniceftunisia/interactor/ChildJsonFormInteractorTest.java

```diff
@@ -0,0 +1,23 @@
+package org.smartregister.uniceftunisia.interactor;
+
+import com.vijay.jsonwizard.constants.JsonFormConstants;
+import com.vijay.jsonwizard.interactors.JsonFormInteractor;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ChildJsonFormInteractorTest {
+
+    @Test
+    public void testChildFormInteractorMapContainsCustomEditTextWidget() {
```

Comment:
i guess the map will always have EDIT_TEXT event without extending ... you can add a check for instance type

Reply:
Noted. Correction made